src/components/navBar/navBar.module.scss

```diff
@@ -79,4 +78,106 @@ $theme-active: #87d870;
         color: $white;
         font-weight: 600;
     }
+
+    .hamburgerIcon {
+        display: none;
+    }
+    ul li {
+        list-style: none;
+    }
+    .navLinks ul {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 1rem;
+    }
+    .active {
+        color: $theme-active;
+        text-decoration: underline;
+        text-decoration-thickness: $thickness;
+        text-underline-offset: $offset;
+    }
+}
+@media (max-width: 970px) {
+    .navBar {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        position: relative;
+        .navLinks {
+            a {
+                padding: 10px 40px;
+                text-align: left;
+                color: $theme-blue;
+                display: none;
+            }
+        }
+    }
+
+    .signInLink {
+        position: absolute;
+        top: 10px;
+        right: 20px;
+    }
+
+    .showLogo {
+        display: none !important;
+    }
+
+    .hamburgerIcon {
+        display: inline-block !important;
```

Comment:
I still see important being used at multiple places

Reply:
!important here at hamburgerIcon class is necessary, if I remove this then the hamburger Icon is getting removed and Navbar is not looking good, rest I have removed important from everywhere.